### ISSUE TITLE
[FEATURE] External Storage integrations

### ISSUE THREAD

**Issue by SalahAdDin** [frappe/erpnext#41128]
**Is your feature request related to a problem? Please describe.**
Not all small companies have the budget for paying either local storage or Amazon S3, so we begin working with cheaper CDNs like Cloudflare or Cloudinary.

Right now, Frappe only works with local storage and has an [S3 integration](https://github.com/frappe/storage_integration) plugin, but no other possible ways to integrate storage.

**Describe the solution you'd like**
It would be nice to have an official API or application for universal storage, or, at least, for Cloudinary storage.

**Describe alternatives you've considered**

**Additional context**
We are a small company and we are testing ERP Next with docker in a local machine before hosting a VPS online.

**Comment by oleynikd** [comment]
@cwerner1 @SalahAdDin I don't think this [S3 integration](https://github.com/frappe/storage_integration) plugin is still relevant. The repo was archived and app removed from Marketplace.
Is there any way to use S3 storage as a main storage for ERPNext? Not just backups, but as a main storage driver?
Thanks.
